integration/tests/main/03-create-private-package-repository.spec.js

```diff
@@ -121,12 +136,22 @@ test("Create a new private package repository successfully", async ({ page }) =>
 
   // Check upgrade result
   await page.waitForSelector(".left-menu");
-  await expect(page.locator(".left-menu")).toContainText("Up to date");
-  await page.waitForSelector("css=.application-status-pie-chart-number >> text=1");
-  await page.waitForSelector("css=.application-status-pie-chart-title >> text=Ready");
+  // Wait for the app to be deployed and select it from "Applications"
+  await expect(page.locator(".left-menu")).toContainText("Up to date", { timeout: deployTimeout });
+  await page.waitForTimeout(5000);
+  await page.click('a.nav-link:has-text("Applications")');
+  await page.waitForTimeout(3000); // Sometimes typing was too fast to get the result shown
+  await page.locator("input#search").fill(appName);
+  await page.waitForTimeout(3000);
```

Comment:
Perhaps in the future we can fine-tune this timeouts, or even remove them completely.

Reply:
I don't like those timeouts either, they came later in the process due to necessity. Without them, sequential execution of e.g.
```
  await page.click('a.nav-link:has-text("Applications")');
  await page.locator("input#search").fill(appName);
  ```
sometimes would not produce any search result (and the app was there for sure).
Same happened in other combination of instructions where, I would say, frontend was unable to handle the speed of the test execution.